kategory/src/test/kotlin/kategory/data/Function1Test.kt

```diff
@@ -0,0 +1,25 @@
+package kategory
+
+import io.kotlintest.KTestJUnitRunner
+import io.kotlintest.matchers.shouldBe
+import org.junit.runner.RunWith
+
+@RunWith(KTestJUnitRunner::class)
+class Function1Test : UnitSpec() {
+    init {
+        testLaws(MonadLaws.laws(Function1.monad<Int>(), object : Eq<HK<Function1.F, Int>> {
```

Comment:
Since this a new instance of `MonadReader` and so far we have Kleisli and this one I think it's time to add the missing `MonadReader` laws and apply them both here and in `Kleisli`, fine if addressed in a separate PR.